tradeexecutor/state/trade.py

```diff
@@ -278,6 +278,26 @@ def __post_init__(self):
     def strategy_cycle_at(self):
         """Alias for oepned_at"""
         return self.opened_at
+    
+    @property
+    def fee_tier(self):
```

Comment:
Add return value type decorators here, so that Python editors can autocomplete values when editing the code